package.json

```
@@ -14,7 +14,8 @@
     "✅ Do add `devDependencies` that are unique to this service."
   ],
   "dependencies": {
-    "codeforlife": "2.12.2"
+    "codeforlife": "2.12.2",
+    "react-resizable-panels": "^3.0.6"
```

please use exact version numbers for more reliable behavior. we have a bot which scans the packages (snyk) and tells us when to upgrade.
```suggestion
    "react-resizable-panels": "3.0.6"
```